src/backend/feed/worker.js

```diff
@@ -1,19 +1,48 @@
-const feedparser = require('./parser');
 const feedQueue = require('./queue');
+const feedParser = require('./parser');
+const extractUrls = require('../utils/extract-urls');
+const storage = require('../utils/storage');
+// const logger = require('./../utils/logger');
 
 exports.workerCallback = async function(job) {
   const { url } = job.data;
-  const posts = await feedparser(url);
```

Comment:
Can you rebase on `master` please?  We have updated the way this `workerCallback` does its thing.

Reply:
Will do!